cosmos-sdk/codec/skd_providers.go

```diff
@@ -0,0 +1,17 @@
+package codec
```

Comment:
Inside juno we need the sdk-codec, with this when we implement a parser a `ProtoCodecProvider` can be set using the `ProtoCodecProvider` and used inside juno.